public/js/export.js

```diff
@@ -39,49 +39,57 @@ $(function() {
     });
 
     $("#data-creds").dialog({
-            autoOpen : false,
-            modal : true,
-            open : function() {
+        autoOpen : false,
+        modal : true,
+        open : function() {
+            checkInputs();
+            $("#data-creds").keyup(function(e) {
                 checkInputs();
-                $("#data-creds").keyup(function(e) {
-                    checkInputs();
-                    if (e.keyCode == $.ui.keyCode.ENTER && checkInputs()) {
-                        $(this).next().find("button:eq(0)").trigger("click");
-                    }
-                });
-            },
-            buttons : {
-                "Ok" : function() {
-                    $(this).next().find("button:eq(0)").button("disable");
-                    $('#error-messages').empty();
-                    sendData(btoa($("#user").val() + ":" + $("#pass").val()), $("#link").val());
-                },
-                "Cancel" : function() {
-                    $(this).dialog("close");
+                if (e.keyCode == $.ui.keyCode.ENTER && checkInputs()) {
+                    $(this).next().find("button:eq(0)").trigger("click");
                 }
+            });
+        },
+        buttons : {
+            "Ok" : function() {
+                $(this).next().find("button:eq(0)").button("disable");
+                $('#error-messages').empty();
+                sendData(btoa($("#user").val() + ":" + $("#pass").val()), $("#link").val());
+            },
+            "Cancel" : function() {
+                $(this).dialog("close");
             }
-        });
+        }
+    });
 });
 
 function download() {
     var data = "";
     // get feature information
-    data += getFeatureTags().join(" ") + "\n";
+    links = getFeatureLinks($(this));
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "links" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=javascript%3AS2703)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') "getFeatureLinks" declared at line 279 expects 0 arguments, but 1 was provided. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=javascript%3AS930)